optuna/importance/_fanova/_tree.py

```diff
@@ -120,19 +121,24 @@ def _get_marginalized_statistics(self, feature_vector: numpy.ndarray) -> Tuple[f
                     continue
 
                 # If subtree starting from node splits on an active feature, push both child nodes.
-                if (active_features & self._subtree_active_features[node_index]).any():
+                if any(self._subtree_active_features[node_index][active_features].tolist()):
```

Comment:
We can probably add a comment on why we use `any` instead of `numpy.ndarray.any`.